lib/src/_shared/PickerToolbar.tsx

```diff
@@ -95,16 +99,12 @@ const PickerToolbar: React.SFC<PickerToolbarProps> = ({
         {children}
         <IconButton
```

Comment:
Probably we need to also expose props that passing directly to this component.

`MobileKeyboardButtonProps`

Reply:
Okay. At that point, maybe we should remove `mobileKeyboardViewButtonClassName`? To me it seems like it's just yet another prop on that component -- or are `className`s worth treating explicitly in general?

The `onClick={toggleMobileKeyboardView}` is probably worth keeping as a separate prop, as it must be implemented by the user of the component.

Reply:
Ah well, `mobileKeyboardViewButtonClassName` is used by all the Toolbar implementations, so let's keep it.

Reply:
So you proposed "MobileKeyboardButton". I called it "MobileKeyboardViewButton" so far. Now I realised that there's already "get**MobileKeyboardInputViewButton**Text = defaultGet**KeyboardInputSwitchingButton**Text" in the code. Probably we shouldn't have 4 different terms for the same button.

Currently, the [DateTimePicker](https://next.material-ui-pickers.dev/demo/datetime-picker) also shows this button on desktop (because it uses the toolbar in its popover, because it has so many different views). Is that desired and will stay like that (then I'd propose to not have "mobile" in the name of this button), or should the desktop DateTimePicker not have the button (then I can send a separate PR to hide it there).

Reply:
@dmtrKovalenko @oliviertassinari Any opinions on whether the view switching button should be shown on desktop at all?

And what's the proper name for this button?